tests/encoder_test.py

```diff
@@ -19,6 +24,45 @@ def test_encode_decode(self):
                 np.sort(vertices_decoded[faces_decoded].flat)
             )
         )
+    
+    def test_mesh_generation(self):
+        vertices = np.loadtxt(os.path.join(dir_path, 'test_data/sphere_vertices.npy')).astype(np.uint32)
+        faces = np.loadtxt(os.path.join(dir_path, 'test_data/sphere_faces.npy')).astype(np.uint32)
+
+        offsets = 0
+        if os.path.isdir("./meshdir"):
```

Comment:
Please use a temporary file or directory. If this test fails for some reason it will pollute the github repository. 
Python provides a convenient API for this: 
https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory